에러 원인( 해결하기 위해 아래 리스트의 것들을 모두 확인해야 한다.)
1. Setting파일에 모델이 저장된 앱이 installed app에 추가되어 있는지 확인한다.



2. 씨드 데이터 문제
 - 각 데이터별 앱 이름, 모델 이름, 필드 이름이 원래 모델에서 정의한 내용과 같은지 확인
"pk": 1,

"model": "movies.genre",
            앱이름.모델이름
"fields": {"name": "가족"
}
            필드이름 : 필드값



3. 서버에 정의된 모델 변수명 클래쓰 명 확인

※ 변수도 클래쓰도 이름 문제가 없다면 
python manage.py makemigrations
python manage.py migrate
확인

